src/sanitize/sanitizeAssets.js

```diff
@@ -13,6 +14,9 @@ export function sanitizeAssets(assets) {
       ? sanitizePublication(publication)
       : undefined
 
+    //@nacho TODO: revisit- this
+    newAsset.bid = bids ? sanitizeBid(bids) : undefined
```

Comment:
I don't get this line. `bids` becomes `bid` ? Shouldn't it be
```suggestion
    newAsset.bids = bids ? sanitizeBids(bids) : []
```

`sanitizeBid` only omits an `object`'s properties

Reply:
👍 Removed.  I'm not using bids inside any asset